docs/book/tutorials/kubernetes-on-vsphere-with-kubeadm.md

```diff
@@ -1,58 +1,137 @@
-# Running a Kubernetes Cluster on vSphere with kubeadm
-
+# Deploying a Kubernetes Cluster on vSphere with CSI and CPI
+The purpose of this guide is to provide the reader with step by step instructions on how to deploy Kubernetes on vSphere infrastructure. The instructions use kubeadm, a tool built to provide best-practice “fast paths” for creating Kubernetes clusters. The reader will also learn how to deploy the Container Storage Interface and Cloud Provider Interface plugins for vSphere specific operations.
 ## Prerequisites
+This section will cover the prerequisites that need to be in place before attempting the deployment.
+## vSphere requirements
+vSphere 6.7U3 is a prerequisite for using CSI and CPI at the time of writing. This may change going forward, and the documentation will be updated to reflect any changes in this support statement.
+## Recommended Guest Operating System
+VMware recommends that you create a virtual machine template using Guest OS Ubuntu 18.04.1 LTS (Bionic Beaver) 64-bit PC (AMD64) Server. Check it out on VMware PartnerWeb here. This template is cloned to act as base images for your Kubernetes cluster. For instructions on how to do this, please refer to the guidance provided in this blog post by Myles Gray of VMware. Ensure that SSH access is enabled on all nodes. This must be done in order to run commands on both the Kubernetes master and worker nodes in this guide.
```

Comment:
Links missing in two places. Partnerweb and my OS setup tutorial.

Reply:
Add note that if provisioning through my tutorial - they will need to have a SSH public key on their machine and add it to the template (covered in the article, but good to reinforce)

Reply:
URLs have now been added.